app/categories/__init__.py

```diff
@@ -23,5 +24,6 @@
 bp.register_blueprint(send_bp)
 bp.register_blueprint(public_bp)
 bp.register_blueprint(benefits_bp)
+bp.register_blueprint(x_cat_bp)
```

Comment:
Might need some help understanding the naming here and how anti-social behaviour fits into this.

Reply:
Hi @digitalali-moj I named it this way to match the naming in the specification.

X-Cat refers to pages that can be found across multiple categories of law.